frontend/src/utils/index.js

```diff
@@ -1,10 +1,9 @@
 import path from 'path';
-import saveSvgAsPng from 'save-svg-as-png';
```

Comment:
If you stop to use `save-svg-as-png`, you should run `npm un save-svg-as-png`.

Reply:
Thank you. I executed `npm un save-svg-as-png` .